docs/DEVGUIDE.md

```diff
@@ -45,7 +45,7 @@ To clone the repository:
 First `cd` to the root of the cloned repository tree.
 To build the service-catalog you have two options:
 * `make build`
-* `DOCKER=1 make build`
+* `build/run.sh make build`
```

Comment:
To be clear, I'm not against having a "run.sh" script available that throws people into a container so they can run random commands, but I would prefer to require it just to build.